Automattic/sensei
# Add notice asking user for reviews/feedback

### Changes proposed in this Pull Request

* Add option `sensei_installed_at` to save when Sensei was installed/updated for the first time;
* Add condition `installed_since` to show notice only after a given time was passed; (I accept feedback regarding this name)
* Add "interactive" dismissible notice asking the user for reviews if some time passed;
* Add way to run extra "tasks" on the frontend like show/hide a notice when clicking on a button/link in a notice;

### Testing instructions

1. Install the plugin based on this branch of the code;
2. Verify that you can visit the Sensei Home page without any issues;
3. Change the URL on the class `Sensei_Home_Remote_Data_API` to point to a WP with this plugin correctly installed 12-gh-Automattic/senseilms-com-plugins 
4. Alter the option `sensei_installed_at` to have a unix timestamp from more than 10 days ago;
5. Verify if the notice asking for reviews appears;
6. Verify that, if you click "Yes", the notice ask the user for a review;
7. Verify that, if you click "No", the notice will ask the user for feedback;
8. Verify that you can dismiss the notice;
9. Verify if the notice, after dismissed, still appears to other site admins;
10. Verify that the notice won't show for users that are not super admins;

### Screenshots

After the user installed Sensei for some time:

![Image showing notice asking the user if it likes Sensei](https://user-images.githubusercontent.com/529864/207099538-1ff118c5-3a6f-475f-acbc-2ec632d6c306.png)

Image showing notice after user clicked on "Yes" option:

![Image showing notice after user clicked on "Yes" option](https://user-images.githubusercontent.com/529864/207099791-5d083016-4f3c-4a38-8bc9-65268b1efd93.png)


Image showing notice after user clicked on "No" option:

![Image showing notice after user clicked on "No" option](https://user-images.githubusercontent.com/529864/207099645-bc1e2bfa-0bc2-44d1-8eda-f1b832eee084.png)
